### DISCUSSION TITLE
Artisan for probatone 25 typ2

### DISCUSSION THREAD

**Discussion by lau1984** [artisan-roaster-scope/artisan#838] [Q&A]
Hi

I am trying to run artisan connected via ethernet cable to our probatone 25 type 2.

I have been informed by probat that I have the wrong version installed (v2.4.6)

I believe the version I need is v1.1.0 but I am not sure as I can't install it, I have downloaded artisan-1.1.0 folder but can't install the program.

Any help would be greatly appreciated
Lau

**Comment by MAKOMO** [comment]
Please ignore the information from Probat and install v2.4.6 which runs fine with the Probat Probatone 25 Series II if it has the software option installed. You need to put your laptop on the same IP network as the machine and run the Probat Probatone 7" setup under menu Config >> Machines. Artisan will ask for the machines IP address.

**Reply by lau1984** [reply]
I have tried running to he latest version

It does ask for the IP network

At first glance it seems like it's all good but it drops out and a modbus communication error occurred.
![IMG20220302103515](https://user-images.githubusercontent.com/101178977/157215355-91046409-50fd-4539-ad20-a0ff8c1589b7.jpg)
![IMG20220302103531](https://user-images.githubusercontent.com/101178977/157215408-8100b5c0-78ba-4cf5-8a32-9fd00e67e1a2.jpg)

**Comment by MAKOMO** [comment]
Not the latest version. I read  2.1.2 on your screen. Please install v2.4.6 and then run the machine configuration under menu Config as I told you. During this setup it will ask for the IP.

**Reply by lau1984** [reply]
Hi Marko

Thank you for your response

Here are some images with the latest version of artisan after config>machine>probat>probatone 7
![IMG20220309083133](https://user-images.githubusercontent.com/101178977/157329012-d0ff6242-d79c-474f-b074-2ea6bbc3c29d.jpg)
![IMG20220309083151](https://user-images.githubusercontent.com/101178977/157329046-16102098-bfeb-4b2b-aec8-220733783b46.jpg)

**Reply by poundy** [reply]
Just checking, do you know what Marko meant by "must be on the same IP network" ? If so, can you confirm the IP address you used and subnet mask, for your PC, and the IP address you used in the machine setup?  If not, this is your problem.... Without setting your PC's IP address so that it can communicate to the roaster, you won't get any IP communication.

**Reply by lau1984** [reply]
Hi poundy, I think so, but I am not really tech savvy!

The roaster is connected to the laptop via ethernet cable and I am using the IP number that appears on the probat software (pilot roaster shop)

The probat communicates with this software but not with artisan, am I missing something here?

Thanks for taking some time to help.

**Reply by poundy** [reply]
Sorry, I am not familiar with Probats, except on the periphery. Can you elaborate - do you have some Probat software on the same PC as you have Artisan, and it works correctly, but Artisan does not? 

Fundamentally, the PC and the Probat roaster needs to be on the same "routing" network. That means you normally will need to have the IP address of your PC to be set into the same range as the roaster. That means you should tell us what your PC's IP address detaios are (including subnet mask) and what your roaster's IP address is, so that we can make sure these settings line up correctly.
